site/en/blog/renderingng/index.md

```diff
@@ -0,0 +1,466 @@
+[D3D12](https://docs.microsoft.com/en-us/windows/win32/direct3d12/directx-12-programming-guide), and
+[Metal](https://developer.apple.com/metal/): lower-level APIs that use GPUs more efficiently than OpenGL.
+- More composited animations:
+[SVG](/blog/hardware-accelerated-animations/), background color.
+
+Upcoming features unblocked by RenderingNG that we're excited about include:
+
+- [Scroll-linked animations](https://drafts.csswg.org/scroll-animations-1/).
+- [Hidden, yet searchable and accessible DOM](https://github.com/whatwg/html/issues/6040).
+- [Shared element transitions](https://github.com/WICG/shared-element-transitions).
+- [Custom layout](https://www.w3.org/TR/css-layout-api-1/).
+- Off-main-thread compositing; decoupling threading and compositing (this one will get its own follow-on blog post!)
+
+## Key projects that make up RenderingNG
+
+Below is a list of the key projects within RenderingNG. Subsequent blog posts will deep-dive into each of them.
+
+### CompositeAfterPaint
+
+Disentangles compositing from style, layout and paint,
+allowing much-improved reliability and predictable performance, increased throughput,
+and using less memory without sacrificing performance.
+
+### LayoutNG
+
+A ground-up rewrite of all layout algorithms,
+for greatly improved reliability and more predictable performance.
+
+### BlinkNG
+
+A systematic cleanup and refactoring of the Blink rendering engine into cleanly separated pipeline phases.
+This allows for better caching, higher reliability,
+and re-entrant / delayed-rendering features such as content-visibility and container queries.
+
+### GPU Acceleration everywhere
+
+A long-term effort to roll out GPU rasterization, draw and animation on all platforms, all of the time.
+GPU acceleration provides an enormous speedup for most content,
+because every pixel can be processed in parallel.
+It is also an effective method for improving performance on low-end devices, which tend to still have a GPU.
+
+### Threaded scrolling, animations, and decode
+
+A long-term effort to move all scrolling, non-layout-inducing animations,
+and image decoding to the compositor thread.
+
+### Viz
+
+A centralized raster and draw process for Chromium that increases throughput,
+optimizes memory, and allows optimal use of hardware capabilities.
+It also has other benefits less visible to web developers but very visible to users,
+such as unblocking Site Isolation and decoupling the rendering pipeline from browser UI rendering.
+
+### Threaded and accelerated canvas rendering
+
+This is the project that put in place the architectural pieces that made OffscreenCanvas possible.
+
+<table class="width-full">
+  <thead>
+    <tr>
+      <td></td>
+      <th>2011</th>
+      <th>2014</th>
+      <th>2015</th>
+      <th>2016</th>
+      <th>2017</th>
+      <th>2018</th>
+      <th>2019</th>
+      <th>2020</th>
+      <th>2021</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <th>CompositeAfterPaint</th>
```

Comment:
I have a new idea. Let's instead take each row of this table and render it in a column instead of a row, and put it inline next to the project it's for. e.g. for CompositeAfterPaint we'd have a table with two columns and 6 rows.

Column 1: year
Column 2: things achieved in that year

Reply:
I've had a go at this in the latest commit. The options we have for tables are here https://developer.chrome.com/docs/handbook/components/#tables